Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -1937,6 +1937,8 @@ void RenderStyle::conservativelyCollectChangedAnimatableProperties(const RenderS
             changingProperties.m_properties.set(CSSPropertyAnchorName);
         if (first.positionAnchor != second.positionAnchor)
             changingProperties.m_properties.set(CSSPropertyPositionAnchor);
+        if (first.positionArea != second.positionArea)
```

Comment:
You need to do something for RenderStyle::diff() too (presumably you'd trigger layout).